packages/inferno/src/DOM/props.ts

```diff
@@ -14,25 +15,21 @@ function createLinkEvent(linkEvent, nextValue) {
 }
 
 export function patchEvent(name: string, nextValue, dom) {
-  const nameLowerCase = name.toLowerCase();
+  const event = name.substr(2).toLowerCase();
```

Comment:
Lets refactor this into common.ts uti, there is identical code in delegation.ts

https://github.com/infernojs/inferno/blob/master/packages/inferno/src/DOM/events/delegation.ts#L85-L87

Reply:
Good idea! 👍